ts/parts/Axis.ts

```diff
@@ -149,14 +149,14 @@ declare global {
             zIndex?: number;
         }
         interface XAxisDateTimeLabelFormatsOptions {
-            day?: XAxisDateTimeLabelFormatsSubOptions;
-            hour?: XAxisDateTimeLabelFormatsSubOptions;
-            millisecond?: XAxisDateTimeLabelFormatsSubOptions;
-            minute?: XAxisDateTimeLabelFormatsSubOptions;
-            month?: XAxisDateTimeLabelFormatsSubOptions;
-            second?: XAxisDateTimeLabelFormatsSubOptions;
-            week?: XAxisDateTimeLabelFormatsSubOptions;
-            year?: XAxisDateTimeLabelFormatsSubOptions;
+            day?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            hour?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            millisecond?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            minute?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            month?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            second?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            week?: (string|XAxisDateTimeLabelFormatsSubOptions);
+            year?: (string| XAxisDateTimeLabelFormatsSubOptions);
         }
         interface XAxisDateTimeLabelFormatsSubOptions {
```

Comment:
I see this is an existing interface, but I think the name is too specific. It could be generalized to `DateTimeLabelFormatOptions`

Reply:
Renamed and moved.